about.html

```diff
@@ -0,0 +1,152 @@
+          <li><a class="nav-link" href="">SPONSOR</a></li>
+          <li><a class="nav-link" href="">NEWS</a></li>
+          <li><a class="campaign nav-link" href="">CAMPAIGN</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section class="header">
+      <div class="background">
+        <nav>
+          <i class="fa fa-bars menu"></i>
+        </nav>
+        <div class="hello">
+          <span class="span-hello">"HELLO! AFRICAN WOMEN"</span>
+        </div>
+        <h1 class="about-h1">
+          EMPOWERING AFRICAN WOMEN <br />
+          GLOBAL SUMMIT <br />
+          2023
+        </h1>
+        <p class="description">
+          A joyful celebration believing in the value of empowering women to
+          achieve their destiny by removing barriers that hold them back with
+          people from over 54 countries is taking place in April, Ethiopia.
+        </p>
+
+        <div class="queries">
+          For queries please contact at info@AWEConference.org
+        </div>
+      </div>
+    </section>
+
+    <section class="empowering">
+      <h2>EMPOWERING AFRICAN WOMEN <br />conference discussion issues</h2>
+      <hr class="empowering-hr" />
+
+      <ul class="lists">
+        <li class="list">
+          1. Place women as leaders and give them decision making roles
+        </li>
+        <li class="list">2. More Job Opportunities for Women</li>
+        <li class="list">
+          3. Invest in Women’s Entrepreneurial Ideas, Emotionally and
+          Financially
+        </li>
+        <li class="list">4. Taking Action against Unpaid Labour Work</li>
+        <li class="list">5. Mentoring Women Professionally and Personally</li>
+      </ul>
+    </section>
+
+    <section class="conference">
+      <h3>See the past EMPOWERING AFRICAN WOMEN conference</h3>
+      <hr class="conference-hr" />
+      <div class="col">
+        <div class="events">
+          <div class="absolute">
+            <h2>2023</h2>
+            <p>Conference</p>
+          </div>
+          <div class="overlay"></div>
+          <img
+            class="event-pic"
+            src="/images/Awc-1.jpg"
+            alt="Empowering Africa women conference"
+          />
+        </div>
+        <div class="events">
+          <div class="absolute">
+            <h2>2023</h2>
+            <p>Conference</p>
+          </div>
+          <div class="overlay"></div>
+          <img
+            class="event-pic"
+            src="/images/AWC-2.jpg"
+            alt="African women speaking at confrance"
+          />
+        </div>
+      </div>
+    </section>
+
+    <footer class="about-footer">
+      <div>
+        <img
+          class="footer-image"
+          src="./images/logo.png"
+          alt="Empowering African women log"
+        />
+      </div>
+      <p class="footer-detail">
+        © 2022 International Conference ,African Women Empowerment <br />Some
+        Rights Reserved.
+      </p>
+    </footer>
```

Comment:
- [ ] Please add the "Partner" section to the mobile version of the "About" page to match the given design.
- [ ] Kindly make sure that the footer section in the "About" page has a different background color on the mobile view than the one you used.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/233106248-035bda5d-86fe-4a62-b4b5-0fab44b65f72.png) | ![image](https://user-images.githubusercontent.com/62290545/233106171-cbe1b908-c1a5-4198-8bf3-d4bc6bd13e8a.png) |